database/migrations/2020_03_25_201407_add_contact_vcard_data.php

```diff
@@ -0,0 +1,32 @@
+<?php
+
+use Illuminate\Support\Facades\Schema;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Database\Migrations\Migration;
+
+class AddContactVcardData extends Migration
+{
+    /**
+     * Run the migrations.
+     *
+     * @return void
+     */
+    public function up()
+    {
+        Schema::table('contacts', function (Blueprint $table) {
+            $table->string('vcard', 65536)->after('gravatar_url')->nullable();
```

Comment:
What does this do? It’s not clear.

Reply:
The best explanation is here:
https://sabre.io/dav/building-a-carddav-client/#retain-full-vcards%21

Reply:
The idea is to record the generated vcard.
It will only be used for external address books. Internal default address book will not save the vcard.